client/websocket/websocket.go

```diff
@@ -352,7 +352,7 @@ func wsLoadCandles(s *Server, cl *wsClient, msg *msgjson.Message) *msgjson.Error
 func wsUnmarket(_ *Server, cl *wsClient, _ *msgjson.Message) *msgjson.Error {
 	cl.feedMtx.Lock()
 	defer cl.feedMtx.Unlock()
-	if cl.feed.loop != nil {
+	if cl.feed != nil && cl.feed.loop != nil {
```

Comment:
I think the mistake here was the `.loop` part.  Looks like it was meant to be just `if cl.feed != nil {`.  (See `connect` and `loadMarket`).  If `cl.feed` is non-nil, then it's good to use it.

Reply:
Yeah, you're right. I've double-checked it, and examining `cl.feed` is enough here.